### PR TITLE
Vendor install.sh bash style fixes

### PR DESCRIPTION
**Pull request by hyuraku**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew typecheck` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

-----
related https://github.com/Homebrew/brew/pull/11178

### CODE REVIEW

File: Library/Homebrew/cmd/vendor-install.sh [hunk 1]
```diff
@@ -3,41 +3,44 @@
 #:
 #:  Install Homebrew's portable Ruby.
 
-source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+# HOMEBREW_CURLRC, HOMEBREW_LIBRARY, HOMEBREW_STDERR is from the user environment
+# HOMEBREW_CACHE, HOMEBREW_CURL, HOMEBREW_LINUX, HOMEBREW_LINUX_MINIMUM_GLIBC_VERSION, HOMEBREW_MACOS, HOMEBREW_MACOS_VERSION_NUMERIC, HOMEBREW_PROCESSOR is set by brew.sh
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/vendor-install.sh:7]:
```suggestion
# HOMEBREW_CACHE, HOMEBREW_CURL, HOMEBREW_LINUX, HOMEBREW_LINUX_MINIMUM_GLIBC_VERSION, HOMEBREW_MACOS,
# HOMEBREW_MACOS_VERSION_NUMERIC and HOMEBREW_PROCESSOR are set by brew.sh
```

File: Library/Homebrew/cmd/vendor-install.sh [hunk 2]
```diff
@@ -90,56 +93,55 @@ fetch() {
...
+  [[ -n "${HOMEBREW_QUIET}" ]] || ohai "Downloading ${VENDOR_URL}" >&2
+  if [[ -f "${CACHED_LOCATION}" ]]
   then
-    [[ -n "$HOMEBREW_QUIET" ]] || echo "Already downloaded: $CACHED_LOCATION" >&2
+    [[ -n "${HOMEBREW_QUIET}" ]] || echo "Already downloaded: ${CACHED_LOCATION}" >&2
   else
-    if [[ -f "$temporary_path" ]]
+    if [[ -f "${temporary_path}" ]]
     then
       # HOMEBREW_CURL is set by brew.sh (and isn't mispelt here)
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/vendor-install.sh:123]: This comment can also be removed if the disable below can be

File: Library/Homebrew/cmd/vendor-install.sh [hunk 3]
```diff
@@ -3,41 +3,44 @@
...
+# HOMEBREW_CACHE, HOMEBREW_CURL, HOMEBREW_LINUX, HOMEBREW_LINUX_MINIMUM_GLIBC_VERSION, HOMEBREW_MACOS, HOMEBREW_MACOS_VERSION_NUMERIC, HOMEBREW_PROCESSOR is set by brew.sh
+# shellcheck disable=SC2154
+source "${HOMEBREW_LIBRARY}/Homebrew/utils/lock.sh"
 
-VENDOR_DIR="$HOMEBREW_LIBRARY/Homebrew/vendor"
+VENDOR_DIR="${HOMEBREW_LIBRARY}/Homebrew/vendor"
 
 # Built from https://github.com/Homebrew/homebrew-portable-ruby.
 #
 # Dynamic variables can't be detected by shellcheck
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/vendor-install.sh:15]: This comment can also be removed if the disable below can be

**Comment by twocountriestrus** [comment]
Gracias 


Enviado desde Yahoo Mail para iPhone


El viernes, abril 23, 2021, 10:15 a. m., Mike McQuaid ***@***.***> escribió:




@MikeMcQuaid commented on this pull request.

In Library/Homebrew/cmd/vendor-install.sh:
> @@ -3,41 +3,44 @@
 #:
 #:  Install Homebrew's portable Ruby.
 
-source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+# HOMEBREW_CURLRC, HOMEBREW_LIBRARY, HOMEBREW_STDERR is from the user environment
+# HOMEBREW_CACHE, HOMEBREW_CURL, HOMEBREW_LINUX, HOMEBREW_LINUX_MINIMUM_GLIBC_VERSION, HOMEBREW_MACOS, HOMEBREW_MACOS_VERSION_NUMERIC, HOMEBREW_PROCESSOR is set by brew.sh
⬇️ Suggested change-# HOMEBREW_CACHE, HOMEBREW_CURL, HOMEBREW_LINUX, HOMEBREW_LINUX_MINIMUM_GLIBC_VERSION, HOMEBREW_MACOS, HOMEBREW_MACOS_VERSION_NUMERIC, HOMEBREW_PROCESSOR is set by brew.sh
+# HOMEBREW_CACHE, HOMEBREW_CURL, HOMEBREW_LINUX, HOMEBREW_LINUX_MINIMUM_GLIBC_VERSION, HOMEBREW_MACOS,
+# HOMEBREW_MACOS_VERSION_NUMERIC and HOMEBREW_PROCESSOR are set by brew.sh

In Library/Homebrew/cmd/vendor-install.sh:
>    else
-    if [[ -f "$temporary_path" ]]
+    if [[ -f "${temporary_path}" ]]
     then
       # HOMEBREW_CURL is set by brew.sh (and isn't mispelt here)

This comment can also be removed if the disable below can be

In Library/Homebrew/cmd/vendor-install.sh:
>  
-VENDOR_DIR="$HOMEBREW_LIBRARY/Homebrew/vendor"
+VENDOR_DIR="${HOMEBREW_LIBRARY}/Homebrew/vendor"
 
 # Built from https://github.com/Homebrew/homebrew-portable-ruby.
 #
 # Dynamic variables can't be detected by shellcheck

This comment can also be removed if the disable below can be

File: Library/Homebrew/cmd/vendor-install.sh [hunk 4]
```diff
@@ -4,15 +4,13 @@
...
 # HOMEBREW_CURLRC, HOMEBREW_LIBRARY, HOMEBREW_STDERR is from the user environment
-# HOMEBREW_CACHE, HOMEBREW_CURL, HOMEBREW_LINUX, HOMEBREW_LINUX_MINIMUM_GLIBC_VERSION, HOMEBREW_MACOS, HOMEBREW_MACOS_VERSION_NUMERIC, HOMEBREW_PROCESSOR is set by brew.sh
+# HOMEBREW_CACHE, HOMEBREW_CURL, HOMEBREW_LINUX, HOMEBREW_LINUX_MINIMUM_GLIBC_VERSION, HOMEBREW_MACOS,
+# HOMEBREW_MACOS_VERSION_NUMERIC and HOMEBREW_PROCESSOR are set by brew.sh
 # shellcheck disable=SC2154
 source "${HOMEBREW_LIBRARY}/Homebrew/utils/lock.sh"
 
 VENDOR_DIR="${HOMEBREW_LIBRARY}/Homebrew/vendor"
 
-# Built from https://github.com/Homebrew/homebrew-portable-ruby.
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/vendor-install.sh:10]: Leave this comment around.

**Review by MikeMcQuaid — DISMISSED** [review]

File: Library/Homebrew/cmd/vendor-install.sh [hunk 5]
```diff
@@ -119,6 +119,7 @@ fetch() {
   else
     if [[ -f "${temporary_path}" ]]
     then
+      # shellcheck disable=SC2153
```
Comment (MikeMcQuaid) [inline Library/Homebrew/cmd/vendor-install.sh:122]: Can this get a comment too (just explaining it's not a typo and where it's set)

Comment (hyuraku) [reply Library/Homebrew/cmd/vendor-install.sh:122]: Without the comment `shellcheck disable`, the message shows on `GitHub ACtion CI/syntax`
```
In /home/linuxbrew/.linuxbrew/Homebrew/Library/Homebrew/cmd/vendor-install.sh line 122:
11
      "${HOMEBREW_CURL}" "${curl_args[@]}" -C - "${VENDOR_URL}" -o "${temporary_path}"
12
       ^--------------^ SC2153: Possible misspelling: HOMEBREW_CURL may not be assigned, but HOMEBREW_CURLRC is.
```

Comment (MikeMcQuaid) [reply Library/Homebrew/cmd/vendor-install.sh:122]:
```suggestion
      # HOMEBREW_CURL is set by brew.sh (and isn't mispelt here)
      # shellcheck disable=SC2153
```

**Review by MikeMcQuaid — APPROVED** [review]

**Comment by MikeMcQuaid** [comment]
Thanks again @hyuraku!

**Pull request merged**